### PR TITLE
requirements

### PR DESCRIPTION
**Pull request by ashishrudra**
was planning to put the `requirements` in mission show page nested under `deliverable`. but it looks messy.So i took it out as `deliverable` show page. let me what you think

### CODE REVIEW

File: spec/controllers/requirements_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    { name: nil }
+  }
+
+  let(:valid_session) { {} }
+
+  # This should return the minimal set of attributes required to create a valid
+  # Requirement. As you add validations to Requirement, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
```
Comment (stevenweber) [inline spec/controllers/requirements_controller_spec.rb]: This is overwriting #valid_attributes and skipping all the tests.

File: config/locales/en.yml [hunk 2]
```diff
@@ -38,6 +39,23 @@ en:
...
+      edit: (edit)
+      requirements: Requirements
+      requirement_html: "%{name} %{edit_link}"
+
+  requirements:
+    new:
+      title: Add a Requirement
+      description:
+        what needs to be done?
+        Each reuirement should represent an actionable item.
```
Comment (stevenweber) [inline config/locales/en.yml]: Do you think we need tasks as well -- or do you think that'll nest things too deeply?

## Without tasks

* Move apartment
  * Secure new one
    * Appointment 1 @ x
    * Appointment 2 @ x
    * Put down first months' rent
  * Get things to the new one
    * Hire movers
    * Buy boxes
    * Buy bubble-wrap
    * Pack stuff

## With tasks

* Move apartment
  * Secure new one
    * Find new one 
      * Appointment 1 @ x
      * Appointment 2 @ x
    * Pay for it
      * Put down first months' rent
  * Get things to the new one
    * Get supplies
      * Buy boxes
      * Buy bubble-wrap
    * Move things
      * Hire movers
      * Pack

Comment (ashishrudra) [inline config/locales/en.yml]: yeah.. i feel same. lets skip the tasks.. or we can rename `requirement` to `task`

File: spec/controllers/requirements_controller_spec.rb (hunk 1 above)
Comment (ashishrudra) [inline spec/controllers/requirements_controller_spec.rb]: oops.. fixed

**Pull request merged**